TitaniumTask/app/src/main/java/com/sa/testtask/screens/ListActivity.java

```diff
@@ -54,18 +52,31 @@ protected void onCreate(Bundle savedInstanceState) {
         recyclerView.setLayoutManager(new LinearLayoutManager(this));
         recyclerView.setAdapter(adapter);
         showList(imageList);
-        Subscribe();
+        //should subscribe in resume
+        //if on stop will be called all subscriptions will be lost
+        //Subscribe();
+    }
 
+    @Override
```

Comment:
I agree and I know about it, but forgot to change (in MainActivity it is changed)